slither/detectors/functions/external_function.py

```diff
@@ -131,6 +155,14 @@ def _detect(self):  # pylint: disable=too-many-locals,too-many-branches
             # Next we'll want to loop through all functions defined directly in this contract.
             for function in contract.functions_declared:
 
+                # If all of the function arguments are non-reference type or calldata, we skip it.
+                reference_args = []
```

Comment:
Can you include which parameters need to be change for memory to calldata in the result? Right now it only specifies the function name, but it'd be helpful if it said "change `memory myarray` to `calldata myarray` in `function A()`

Reply:
Added in https://github.com/crytic/slither/pull/1022/commits/c878a8fc25b8a82fb41b4400f2c97f0ec6061fc6